### PR TITLE
xonsh 0.9.19

### PR DESCRIPTION
**Pull request by samueldg**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/xonsh.rb [hunk 1]
```diff
@@ -19,31 +19,6 @@ class Xonsh < Formula
...
-  resource "six" do
-    url "https://files.pythonhosted.org/packages/21/9f/b251f7f8a76dec1d6651be194dfba8fb8d7781d10ab3987190de8391d08e/six-1.14.0.tar.gz"
-    sha256 "236bdbdce46e6e6a3d61a337c0f8b763ca1e8717c03b369e87a7ec7ce1319c0a"
-  end
-
-  resource "wcwidth" do
-    url "https://files.pythonhosted.org/packages/25/9d/0acbed6e4a4be4fc99148f275488580968f44ddb5e69b8ceb53fc9df55a0/wcwidth-0.1.9.tar.gz"
-    sha256 "ee73862862a156bf77ff92b09034fc4825dd3af9cf81bc5b360668d425f3c5f1"
-  end
-
```
Comment (samueldg) [inline Formula/xonsh.rb:46]: These deletions look kind of fishy, I don't think it's intended 🤔

**Comment by samueldg** [comment]
Could a reviewer help me troubleshoot this?
(@chenrui333 Looks like you've reviewed the changes to this formula a few times. Sorry for summoning you like this!)

Is there a recent change that could explain why all the extra resources would be automatically removed by `brew bump-formula-pr`? I don't mind adding them back manually, but I'm afraid it would either break something or a check, or just require manual updates down the road every time someone runs `brew bump-formula-pr`.

Worth noting that those extra dependencies are declared as extras: https://github.com/xonsh/xonsh/blob/377ab8d9deaef75bf5bf8323c8c1141f345f4b9a/setup.py#L393-L400 -- maybe that's somehow related?

**Comment by SMillerDev** [comment]
Yeah, you'll need to manually add those back. Can you also make an issue in homebrew/brew for this? 

CC @Rylan12

**Comment by Rylan12** [comment]
Yes, this is a mistake by `bump-formula-pr`. We've added a feature that allows `bump-formula-pr` to update these resources automatically. In some cases, this does so incorrectly (which is what's happening here).

For now, can you manually add the resources back? I'll open a PR to add `xonsh` to the blocklist so this won't happen in the future.

**Comment by samueldg** [comment]
@SMillerDev @Rylan12 I just reverted the resources back in -- see 639aa54.

Thanks for adding `xonsh` to the blocklist as well! Having that list is a good example of foresight 😄

**Comment by Rylan12** [comment]
LGTM. Can you squash your commits into one called `xonsh 0.9.19` (same as the PR name)?

It would be great if you could also add a license line. [It looks like `BSD-2-Clause` to me](https://github.com/xonsh/xonsh/blob/master/license).

```ruby
license "BSD-2-Clause"
```

**Comment by samueldg** [comment]
@Rylan12: Should be all set! 

I did notice however that the license of the Xonsh project seems to include a clause that's not in the "BSD-2-Clause" license, but appears in "BSD-2-Clause-FreeBSD".

Compare:
* https://spdx.org/licenses/BSD-2-Clause.html
* https://spdx.org/licenses/BSD-2-Clause-FreeBSD.html

I grepped in the whole formulas and didn't find any occurrence of the latter though, so I'm not totally sure about adding this.

**Comment by Rylan12** [comment]
Ooh, good catch! We should definitely use the version with the added clause. Don't worry about the lack of use so far; we have support for it but just haven't encountered any formulae that have needed it yet. Licenses are still relatively new to Homebrew so we're still working on adding them for all formulae.

The `BSD-3-Clause-FreeBSD` has been deprecated, though, so we should use [`BSD-2-Clause-Views`](https://spdx.org/licenses/BSD-2-Clause-Views.html) instead.

**Comment by samueldg** [comment]
@Rylan12 Good suggestion! I updated the license to `BSD-2-Clause-Views` and squashed everything.

**Review by Rylan12 — APPROVED** [review]

**Comment by Rylan12** [comment]
Looks good. Thanks for being flexible with the resources and the license, @samueldg!

**Pull request closed without merging**